test/crdsvalidation/kongcacertificate/testcases/required_fields.go

```diff
@@ -0,0 +1,28 @@
+package testcases
+
+import (
+	configurationv1alpha1 "github.com/kong/kubernetes-configuration/api/configuration/v1alpha1"
+	"github.com/samber/lo"
```

Comment:
These should be in separate import groups but let's handle that in a dedicated PR.

PTAL: https://github.com/Kong/kubernetes-configuration/pull/81